package.json

```diff
@@ -50,6 +50,7 @@
     "keycloak-request-token": "^0.1.0",
     "nock": "^9.0.2",
     "phantomjs-prebuilt": "^2.1.14",
+    "chromedriver": "^2.41.0",
```

Comment:
I left this here only for testing purposes, this should be probably removed when this PR is ready to merge.

Reply:
Let's just remove it, so we can focus on a single issue.